coalib/collecting/Collectors.py

```diff
@@ -350,6 +351,17 @@ def get_all_bears_names():
     return [bear.name for bear in get_all_bears()]
 
 
+def get_all_languages():
+    """
+    Get a ``list`` of all languages supported by coala, including
+    ``Unknown`` language.
+    """
+    return Languages([
+        language_file[: -3] for language_file in
+        os.listdir('coalib/bearlib/languages/definitions/')
```

Comment:
They are all here, loaded, so dont look at the directory.

Reply:
How are they loaded until I do something like `from coalib.bearlib.languages import Python` for every language?

Reply:
See `coalib.bearlib.languages.__dict__` for one way to get them all

But manually listing them is also ok. The test will fail if someone forgets to add one.